tka/tka.go

```diff
@@ -23,7 +24,7 @@ var cborDecOpts = cbor.DecOptions{
 	TagsMd:      cbor.TagsForbidden,
 
 	// Arbitrarily-chosen maximums.
-	MaxNestedLevels:  8,
+	MaxNestedLevels:  16, // Most likely to be hit for SigRotation sigs.
```

Comment:
Is this going to be enough?

Reply:
Thinking on it some more, I think we could get away with a hard limit of 3 (direct -> rotation, or credential -> rotation -> rotation, I havent implemented credential yet).